src/fs.js

```diff
@@ -55,17 +46,6 @@ Fs.prototype = {
         if (lookups.hasOwnProperty(b)) {
           var checkDir = path.join(relativeTo, new Array(a).join('../'), b, mod);
           var packageFile = path.join(checkDir, lookups[b]);
```

Comment:
SInce we're doing `for (var b in lookups)`, this is indeterminate as to whether `bower_components` or `node_modules` is looked up first, right? Is that OK?